x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/feature_table/feature_table_expanded_row.tsx

```diff
@@ -63,7 +63,8 @@ export const FeatureTableExpandedRow = ({
       privilegeCalculator.updateSelectedFeaturePrivilegesForCustomization(
         feature.id,
         privilegeIndex,
-        e.target.checked
+        e.target.checked,
+        allSpacesSelected
```

Comment:
__note:__ a new test should fail if you revert this change (or change `allSpacesSelected` to `false` as it was implicitly set before the fix).